### PR TITLE
Updated the PHP CS Fixer config

### PR DESCRIPTION
**Pull request by sanmai**
Updated the PHP CS Fixer config to follow established rules. No code change needed.

Avoids discussions like [this](https://github.com/infection/infection/pull/269#discussion_r177657790) and [this](https://github.com/infection/infection/pull/258#discussion_r177657524).

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -31,6 +31,12 @@ return \PhpCsFixer\Config::create()
         'phpdoc_align' => false,
         'phpdoc_summary' => false,
         'yoda_style' => false,
+        'ordered_class_elements' => [
+            'use_trait',
+            'constant_public',
+            'property_public',
```
Comment (borNfreee) [inline .php_cs.dist]: Does it mean that other settings from this rules are left by default? I'm interested in `'method_public', 'method_protected', 'method_private'` - exactly in this order

Comment (sanmai) [reply .php_cs.dist]: Other settings will be as before: unordered default. 

That's 103 files changed if you want them in this order. And say goodbye to `git blame` along the way. I wouldn't recommend this.

Comment (borNfreee) [reply .php_cs.dist]: got it :(

Comment (BackEndTea) [reply .php_cs.dist]: > That's 103 files changed if you want them in this order. And say goodbye to git blame along the way. I wouldn't recommend this.

If its something we really want, its better to just do it. The longer we wait, the bigger the impact of these things will be.

But if its not that important we can just skip it.

**Comment by borNfreee** [comment]
Thank you @sanmai

**Pull request merged**